openml/_api_calls.py

```diff
@@ -42,7 +43,7 @@ def _perform_api_call(call, request_method, data=None, file_elements=None):
     url += call
 
     url = url.replace('=', '%3d')
-
+    logging.info('[%s] %s' % (request_method, url))
```

Comment:
Please don't use `%` for string interpolation in logging, but pass the formatting arguments as regular arguments to the function as `logging.info('[%s] %s', request_method, url)`. Then, they're only interpolated if the logging level requires the program to do so.